docs/sources/user_guide/runtime_assurance.rst

```diff
@@ -216,3 +217,21 @@ The mathematical formulations used by the RAM rely on an implicit assumption abo
   There exist a fundamental trade-off between conservatism and optimality at the control level. The more certainty one has in things like dynamics, localization, and communication timing, the less conservatism must be introduced to ensure a low probability of collision.
 
   The Supervisor is able to account for uncertainty in an explicit and quantitative way. Please `contact 3laws`_ to learn more about the process of tailoring Supervisor to better handle uncertainties and communication delays in your system.
+
+
+Visualization
+=============
+
+The RAM publishes a set of markers to visualize the processing done by the Safety Maps, and the safety constraints generated by the Kernel Generator:
+
```

Comment:
Please understand that this manual is official documentation for our product, and as such makes representations about the capabilities and guarantees provided by our product.

My understanding is that CBFs are a mathematical construct for which there is a mathematical proof.  The main benefit of this proof is ability to achieve constraint enforcement under specific conditions. The constraints may be related to performance, stability, and predictability; which generally leads to a safer system.  However, the proof itself does not ensure safety.  Safety would be a consequence of proper system operation and of a solid match between the models and the actual system.   

The documentation should be limited in how we imply that our product provides safety.  The terminology used in our manuals need to correspond to common industry understandings and practices rather than the language that was used for papers in an academic/research setting where "safety" is not taken to mean that you can put your child in front of the system and walk away without worry.

Reply:
We'll do our best